v4/source/files.yaml

```
@@ -145,7 +145,16 @@
         "401":
           $ref: "#/components/responses/Unauthorized"
         "403":
-          $ref: "#/components/responses/Forbidden"
+          description: Do not have appropriate permissions
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/AppError"
+          headers:
+            First-Inaccessible-File-Time:
+              schema:
+                type: integer
+              description: This header is included with the value "1" if the post is past the cloud's plan limit.
```

0/5 - I think it would be more correct to say "file" instead of post here. Same applies for the other additions.

```suggestion
              description: This header is included with the value "1" if the file is past the cloud's plan limit.
```